Modules/Devices/src/Devices.Application/Identities/Commands/ChangeFeatureFlags/Validator.cs

```diff
@@ -1,15 +1,20 @@
 ﻿using Backbone.BuildingBlocks.Application.Abstractions.Exceptions;
 using Backbone.Modules.Devices.Domain.Entities.Identities;
 using FluentValidation;
+using Microsoft.Extensions.Options;
 
 namespace Backbone.Modules.Devices.Application.Identities.Commands.ChangeFeatureFlags;
 
 public class Validator : AbstractValidator<ChangeFeatureFlagsCommand>
 {
-    public Validator()
+    public Validator(IOptions<ApplicationConfiguration> configuration)
     {
         RuleForEach(c => c.Keys).Must(k => FeatureFlagName.Validate(k) == null)
             .WithMessage("Invalid feature flag name. Check length and the used characters.")
             .WithErrorCode(GenericApplicationErrors.Validation.InvalidPropertyValue().Code);
+
+        RuleFor(command => command.Count).Must(count => (count <= configuration.Value.MaxNumberOfFeatureFlagsPerIdentity))
```

Comment:
Use existing fluent validation methods wherever possible.
```suggestion
        RuleFor(command => command.Count).LessThanOrEqualTo(configuration.Value.MaxNumberOfFeatureFlagsPerIdentity)
```